docs/conf.py

```diff
@@ -30,19 +30,14 @@
 from docutils.utils import get_source_line
 
 
-def _warn_node(self, msg, node):
-    """Do not warn on external images."""
-    if not msg.startswith('nonlocal image URI found:'):
-        self._warnfunc(msg, '{0}:{1}'.format(get_source_line(node)))
-
-sphinx.environment.BuildEnvironment.warn_node = _warn_node
-
-
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #needs_sphinx = '1.0'
 
+# Do not warn on external images.
```

Comment:
@tiborsimko @nikofil The packages docs generation are already broken as it is, because of the open ended requirement. Hence it's the same amount of effort to fix the problem as to lock it the sphinx version, so let's go ahead an upgrade the Sphinx version.